Application/src/controller/Import.java

```diff
@@ -15,7 +15,7 @@
 import java.util.List;
 
 /**
- * @version 1.0
```

Comment:
Contributor tag

Reply:
Not sure if that part warrents a contributor tag